scripts/us_census/acs5yr/subject_tables/common/resolve_geo_id_test.py

```diff
@@ -0,0 +1,43 @@
+"""Tests for resolve_geo_id."""
+
+import unittest
+from .resolve_geo_id import convert_to_place_dcid
+
+
+class ResolveCensusGeoIdTest(unittest.TestCase):
+
+    def test_county_geoId(self):
+        # Sussex County, Delaware (0500000US10005)
+        self.assertEqual(convert_to_place_dcid("0500000US10005"), 'geoId/10005')
+        # Natchitoches Parish, Louisiana (0500000US22069)
+        self.assertEqual(convert_to_place_dcid("0500000US22069"), 'geoId/22069')
+
+    def test_state_geoId(self):
+        # California
+        self.assertEqual(convert_to_place_dcid("0400000US06"), 'geoId/06')
+
+    def test_city_geoId(self):
+        # Arlington CDP, Virginia(1600000US5103000)
+        self.assertEqual(convert_to_place_dcid("1600000US5103000"),
+                         'geoId/5103000')
+
+    def test_school_district_geoId(self):
+        # Portsmouth City Public Schools, Virginia (9700000US5103000)
+        self.assertEqual(convert_to_place_dcid("9700000US5103000"),
+                         'geoId/sch5103000')
+
+    def test_not_interesting_summary_levels(self):
+        # Dalton, GA Urbanized Area (2010)[400C100US22069]
+        self.assertEqual(convert_to_place_dcid("400C100US22069"), '')
+
+    def test_zip_code(self):
+        # 65203
+        self.assertEqual(convert_to_place_dcid("86000US65203"), 'zip/65203')
+
+    def test_country(self):
+        # USA
+        self.assertEqual(convert_to_place_dcid("0100000US"), 'country/USA')
+
```

Comment:
Can you please add a test for all codes in _SUMMARY_LEVEL_PREFIX_MAP?

A single combined test with map form code to expected resolved id is also fine.
Check that all prefixes in  _SUMMARY_LEVEL_PREFIX_MAP are covered in the test.


example:
test_geo_ids = {
  '040000US1234': 'geoId/1234',
   '0500000US1234': 'geoId/1234',
    '0600000US1234': 'geoId/1234',
    '1400000US1234': 'geoId/1234',
    '1500000US1234': 'geoId/1234', <--- should this be geoId1234? as / is missing
    '1600000US1234': 'geoId/1234',
    '50000US1234': 'geoId/1234',  <--- / missing?
    '8600000US1234': 'zip/1234',
    '95000US1234': 'geoId/sch1234',
    '960US1234': 'geoId/sch1234',
    '970': 'geoId/sch1234',
    '010': 'country/USA'
}

assertEquals(test_geo_ids.keys(), _SUMMARY_LEVEL_PREFIX_MAP.keys())

Please use valid numbers that map to real places instead of '1234' above.
You can get a sample here: https://autopush.datacommons.org/tools/stat-var#statVar=Count_Person_Male

Reply:
Thank you for the suggestion, this makes it easy to add new tests. I have incorporated this change.